java-frontend/src/test/files/se/customresources/FactoryClosed.java

```diff
@@ -4,8 +4,9 @@
 //    factoryMethod = org.sonar.custom.ResourceFactory#createResource(java.lang.String)
 //    closingMethod = org.sonar.custom.GenericResource#closeResource(java.lang.String)
 
+interface OpenedResource {}
 
-public class GenericResource {
+public class GenericResource implements OpenedResource {
```

Comment:
As discussed, we need the interface, because it is used as return type in `ResourceFactory`